### DISCUSSION TITLE
Link skipping problem

### DISCUSSION THREAD

**Discussion by JingZaia** [KevinnZou/compose-webview-multiplatform#130] [Q&A]
Unable to navigate to new page links within the site,
That is, the link opened for the new page can not be opened, click without any jump, what should I do to let her jump

**Comment by KevinnZou** [comment]
@JingZaia Thanks for your feedback! Could you provide a test URL?

**Comment by JingZaia** [comment]
Let me do a Google search for an example.

I use Google to search for Wikipedia, and then Google will find the Wikipedia official website. I want to click on the Wikipedia official website to view the content, but I can't jump to it no matter how hard I click.
If the link does not require opening a new page, it can be opened normally. If the link opens a new page, it cannot be opened normally.
How to solve the problem of link redirection to new pages not being able to open properly

**Reply by KevinnZou** [reply]
@JingZaia Which platform are you using? I tested it locally and it worked well. I searched for Wikipedia using Google and clicked on the result, successfully reaching the Wikipedia website.

**Reply by JingZaia** [reply]
I just realized that only iOS doesn't work, Android jumps normally

**Reply by KevinnZou** [reply]
Which iOS device and system version are you using? It works well on my iPhone 15 pro max.

**Reply by JingZaia** [reply]
iphone 15 version 17.0
My code
![image](https://github.com/KevinnZou/compose-webview-multiplatform/assets/96556562/5a2e89a2-ab67-4004-b9a0-33c881b9209b)
Does iOS require separate property settings to redirect links to new pages？

**Reply by KevinnZou** [reply]
There are no separate settings that need to be configured for iOS. Have you tested it with the native WKWebView?

**Reply by JingZaia** [reply]
![Apr-06-2024 12-03-42](https://github.com/KevinnZou/compose-webview-multiplatform/assets/96556562/28db1a30-f831-4ad9-a89f-b6d6b30d172b)


IOS native not working
Android can run normally, iOS cannot

**Reply by JingZaia** [reply]
Can you test the website juejin.cn? 
Is this a website issue or an iOS adaptation issue

**Reply by KevinnZou** [reply]
juejin.cn did failed. I am not sure how it realizes page jump. But I think it is a website issue since it works well with other websites like github and wikipedia.
